core/src/main/java/org/moditect/commands/GenerateModuleInfo.java

```diff
@@ -345,6 +381,9 @@ private ModuleDeclaration applyOpensPatterns(ModuleDeclaration moduleDeclaration
 
     private ModuleDeclaration parseGeneratedModuleInfo() {
         Path moduleDir = workingDirectory.resolve( autoModuleNameForInputJar );
+        int multiReleaseIdx = jdepsExtraArgs.indexOf("--multi-release");
```

Comment:
Does this need mention in the README?

Reply:
No, it's not needed but this particular modification is currently set to work only with jdeps 14+, it's not compatible with previous versions anymore. I'll modify it to make it compatible with jdeps 9-13.

Reply:
done as well